repos/spack_repo/builtin/packages/py_xarray/package.py

```diff
@@ -81,17 +91,22 @@ class PyXarray(PythonPackage):
     depends_on("py-pandas@1.1:", when="@0.20:", type=("build", "run"))
     depends_on("py-pandas@1.4:", when="@2023.7.0:", type=("build", "run"))
     depends_on("py-pandas@2.0:", when="@2024.7.0:", type=("build", "run"))
+    depends_on("py-pandas@2.2:", when="@2025.7.1:", type=("build", "run"))
 
     depends_on("py-packaging@20:", when="@0.21:", type=("build", "run"))
     depends_on("py-packaging@21.3:", when="@2023.7.0:", type=("build", "run"))
     depends_on("py-packaging@23.1:", when="@2024.7.0:", type=("build", "run"))
+    depends_on("py-packaging@24.1:", when="@2025.7.1:", type=("build", "run"))
 
+    # +io
     # These are the versions that work with numpy@2:
     # https://github.com/pydata/xarray/pull/9136
     depends_on("py-netcdf4@1.7.1:", when="+io ^numpy@2:", type=("build", "run"))
```

Comment:
You can probably delete this line, it's specified correctly in the netcdf4 package recipe

Reply:
agreed